design/labels.go

```diff
@@ -105,3 +105,25 @@ var _ = a.Resource("label", func() {
 		a.Response(d.Conflict, JSONAPIErrors)
 	})
 })
+
+var _ = a.Resource("work_item_labels", func() {
+	a.Parent("workitem")
+
+	a.Action("list", func() {
+		a.Routing(
+			a.GET("labels"),
+		)
+		a.Description("List labels associated with the given work item")
+		a.Params(func() {
+			a.Param("page[offset]", d.String, `Paging start position is a string pointing to
```

Comment:
pagination is not is not implemented, right?

Reply:
Correct, done in https://github.com/fabric8-services/fabric8-wit/pull/1627/commits/91dfe63ac75776edcf1eaa014ffa9fa2e63b5891